static/src/js/util/accessMethods/__tests__/buildAccessMethods.test.js

```diff
@@ -19,38 +32,82 @@ describe('buildAccessMethods', () => {
     }
     const isOpenSearch = false
 
-    const methods = buildAccessMethods(collectionMetadata, isOpenSearch)
+    buildAccessMethods(collectionMetadata, isOpenSearch)
 
-    expect(methods).toEqual({
-      download: {
-        isValid: true,
-        type: 'download'
-      }
-    })
+    expect(buildDownloadMock).toBeCalledTimes(1)
+
+    expect(buildDownloadMock).toBeCalledWith({ items: [{ online_access_flag: true }] }, false)
   })
 
-  test('returns a download access method for open search', () => {
-    const collectionMetadata = {}
-    const isOpenSearch = true
+  describe('when ordering is disabled', () => {
```

Comment:
Ordering being disabled has nothing to do with buildAccessMethods since we moved pulling that info into `buildEsiEcho`, so this test only needs to happen there